frontend/src/lib/utils/candidateContext.ts

```diff
@@ -106,50 +104,50 @@ const loadUserData = async () => {
   user.set(currentUser);
 };
 
-const logOut = async () => {
+async function logOut() {
```

Comment:
This doesn't seem to be an async function. Also change in the `CandidateContext` type.